src/appstartdialog.cpp

```diff
@@ -118,13 +118,12 @@ void mmAppStartDialog::CreateControls()
     itemBoxSizer5->Add(itemButton9, 0, wxGROW | wxALL, 5);
 
     wxButton* itemButton10 = new wxButton(this, wxID_INDEX, _("&Website"));
-    const wxString s = wxString::Format(_("Visit the %s website")
-        , mmex::getProgramName());
+    const wxString s = wxString::Format(_("Visit the MMEX website"));
```

Comment:
Why this change?

Reply:
Thanks for checking. Updated to `Visit MMEX website for the latest news and updates` to be consistent with text shown in status bar when hovering on `Website` option at:
Menu bar, Help, Community, Website